containers/ecr-viewer/src/app/utils/evaluate/fhir-paths.ts

```diff
@@ -314,6 +320,30 @@ const _fhirPathMappings: { [K in FhirPathKeys]: Omit<FhirPath<K>, "name"> } = {
     type: "ValueX",
     path: "Bundle.entry.resource.where(resourceType = 'Patient').maritalStatus",
   },
+  patientNationality: {
+    type: "string",
+    path: "Bundle.entry.resource.where(resourceType = 'Observation').where(code.coding.where(code = '186034007' and system = 'http://snomed.info/sct')).valueCodeableConcept.coding.display",
```

Comment:
```suggestion
    path: "Bundle.entry.resource.where(resourceType = 'Observation').where(code.coding.where(code = '186034007' and system = 'http://snomed.info/sct')).value",
```

Any reason to not use the more generic handlers/formatters here? (and for contry of residence as well)